packages/block-editor/src/components/block-preview/index.js

```diff
@@ -37,7 +37,7 @@ function ScaledBlockPreview( { blocks, viewportWidth, padding = 0 } ) {
 			// If we're previewing a single block, scale the preview to fit it.
 			if ( blocks.length === 1 ) {
 				const block = blocks[ 0 ];
-				const previewElement = getBlockPreviewContainerDOMNode( block.clientId, containerElement );
+				const previewElement = getBlockPreviewContainerDOMNode( block.clientId );
```

Comment:
I believe the BlockPreview component could be shown several times for the same block list causing this change to break? Templates project cc @getdave

Reply:
Could you elaborate? Which block can I test?